### PR TITLE
Improve error-handling of script executor and simplify README

### PR DESCRIPTION
**Pull request by thomasmunduchira**

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -1,21 +1,44 @@
...
+
+    # check if command is a valid option
+    if command not in commands:
+        exit_with_error(f"command {command} is not valid!")
+
     try:
         subprocess.run(commands[args.command], shell=True)
-    except:
-        pass # hush
+    except BaseException:
```
Comment (klanmiko) [inline start.py]: except KeyboardInterrupt

File: start.py [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
-}
+    'deploy': "pm2 start deploy.sh",
+    'save-deps': 'pip3 freeze > requirements.txt'}
 
 parser = argparse.ArgumentParser(description='Manage the Flask App')
-parser.add_argument("command", help="[all | install | run-windows | format | deploy]", type=str)
+parser.add_argument(
+    "command",
+    help="[run-windows | run-other | install | format | deploy | save-deps]",
+    type=str)
```
Comment (klanmiko) [inline start.py]: add option default="run-other"

File: start.py [hunk 3]
```diff
@@ -1,21 +1,44 @@
...
 
 if __name__ == "__main__":
     args = parser.parse_args()
+    command = args.command.strip()  # strip all whitespace
+
+    # check if command is a valid option
```
Comment (klanmiko) [inline start.py]: redundant comment

File: start.py [hunk 4]
```diff
@@ -1,21 +1,44 @@
...
 parser = argparse.ArgumentParser(description='Manage the Flask App')
-parser.add_argument("command", help="[all | install | run-windows | format | deploy]", type=str)
+parser.add_argument(
+    "command",
+    help="[run-windows | run-other | install | format | deploy | save-deps]",
+    type=str)
 
 if __name__ == "__main__":
     args = parser.parse_args()
+    command = args.command.strip()  # strip all whitespace
```
Comment (klanmiko) [inline start.py]: redundant - argparse strips whitespace

File: README.md [hunk 5]
```diff
@@ -21,26 +21,26 @@ source venv/bin/activate (Other)
...
+python3 start.py install
 ```
 
 ## Running
 
 ```bash
-set FLASK_APP=app.py && flask run (or) python3 start.py run-windows (Windows)
-export FLASK_APP=app.py && flask run (or) python3 start.py all (Other)
+python3 start.py run-windows (Windows)
+python3 start.py run-other (Other)
```
Comment (klanmiko) [inline README.md]: python3 start.py

- no arg as default is to run-other

File: start.py (hunk 1 above)
Comment (thomasmunduchira) [reply start.py]: Good call, I'll fix that.

File: start.py (hunk 2 above)
Comment (thomasmunduchira) [reply start.py]: Okay

**Comment by thebearjew** [comment]
You should both consider using a linter or pretty-printer for PEP8 style so time is not wasted on `camelCase` v.s. `snake_case`

**Pull request merged**